app/connectors/MiddleConnector.scala

```diff
@@ -35,6 +37,26 @@ class MiddleConnector @Inject() (http: HttpClient, httpHandler: HttpHandler)(imp
 
   private def url(path: String) = s"$serviceUrl$path"
 
+  def connectToAtsSaFields(
```

Comment:
don't mix testOnly and prod code. Use a new file.

Reply:
Done